### DISCUSSION TITLE
How can I check if binary is present

### DISCUSSION THREAD

**Discussion by bricebou** [spatie/image-optimizer#166] [Q&A]
Using image-optimizer on Linux systems, I'd like to check if optimizers binaries are installed or not but I can't figure how to do this...

Can you give me some advice or piece of code to do that ?

Thanks in advance :)

**Comment by Tyjow** [comment]
Hi guys,

Try to use theses commands :)

which jpegoptim
which optipng
which pngquant
etc..
